docs/conf.py

```diff
@@ -87,3 +88,10 @@
     'undoc-members': None,
     'show-inheritance': None,
 }
+
+
+# -- Options for sphinx.ext.intersphinx --------------------------------------
+
+# This config value contains the locations and names of other projects
+# that should be linked to in this documentation.
+intersphinx_mapping = {'python': ('https://docs.python.org/3', None)}
```

Comment:
This seems pretty magical. How does it know the layout of python.org?

Reply:
figuring this thing out was why yesterday I said
> \<Chipaca> I am *so* happy we went with sphinx for the docs!!!
> * Chipaca sings a ballad to intersphinx

any sphinx doc site has an `objects.inv` file. The intersphinx mapping setting means it'll get those .inv files (just one in this case), and use them to look up stuff.

It really does feel magical. More in [the intersphinx extension docs](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html).

Reply:
Awesome!!